### ISSUE TITLE
Config.yml to allow extra Args for jupyter notebook commands

### ISSUE THREAD

**Issue by nehiljain** [fivetran/great_expectations#1265]
It would be very helpful to allow running adding extra args to config.yml. This is useful for editing great expectation suites with extra params that can be passed to the underlying jupyter notebook process. For example, for dockerizing GE the jupyter command is `jupyter notebook --allow-root --notebook-dir=. --ip=0.0.0.0 --port=8888 --no-browser`. But if I run `great_expectations suite edit npi_small_db_table.critical` then I get an error
```Traceback (most recent call last):
  File "/usr/local/bin/jupyter-notebook", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/jupyter_core/application.py", line 270, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/traitlets/config/application.py", line 663, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/usr/local/lib/python3.7/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/notebook/notebookapp.py", line 1769, in initialize
    self.init_webapp()
  File "/usr/local/lib/python3.7/site-packages/notebook/notebookapp.py", line 1490, in init_webapp
    self.http_server.listen(port, self.ip)
  File "/usr/local/lib/python3.7/site-packages/tornado/tcpserver.py", line 143, in listen
    sockets = bind_sockets(port, address=address)
  File "/usr/local/lib/python3.7/site-packages/tornado/netutil.py", line 168, in bind_sockets
    sock.bind(sockaddr)
OSError: [Errno 99] Cannot assign requested address
```
 
These args should be optional for developers to use if they chose to do so.

**Comment by Aylr** [comment]
This is a good idea. What would you think of an environment variable like `GE_JUPYTER_COMMAND`? This is similar in spirit to the posix `EDITOR` environment variable. This would be a small and tidy PR if you are interested in implementing this!

**Comment by nehiljain** [comment]
@Aylr happy to do the PR. Can you please link me to the relevant docs and/or code files and I can try and implement it. 

GE_JUPYTER_COMMAND env variable makes a lot more sense.

**Comment by nehiljain** [comment]
@Aylr Question about implementation. Should I be adding it to https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/cli/toolkit.py#L218 or to https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/cli/suite.py#L381?

Is there an example of this being done currently?

**Comment by eugmandel** [comment]
@nehiljain You would be modifying the launch_jupyter_notebook method in toolkit.py to check for this new env var. If it finds, use it. Otherwise, default to ["jupyter", "notebook"] (as is currently).

**Comment by Aylr** [comment]
Fixed in #1347 thanks to the nice work of @nehiljain !
